Formula/bazarr.rb

```diff
@@ -0,0 +1,95 @@
+require "language/node"
+
+class Bazarr < Formula
+  include Language::Python::Virtualenv
+
+  desc "Companion to Sonarr and Radarr for managing and downloading subtitles"
+  homepage "https://www.bazarr.media"
+  license "GPL-3.0-only"
+  head "https://github.com/morpheus65535/bazarr.git", branch: "master"
+
+  stable do
+    url "https://github.com/morpheus65535/bazarr/releases/download/v1.1.1/bazarr.zip"
+    sha256 "0a55474e185c7f84246218097af4caae805ed33d18657f37a8a141d2d847e9e3"
+  end
+
+  depends_on "node" => :build
+  depends_on "ffmpeg"
+  depends_on "gcc"
+  depends_on "numpy"
+  depends_on "pillow"
+  depends_on "python@3.9"
```

Comment:
Can we use Python 3.10?

Reply:
Fair point. I started writing this formula long before the migration to Python 3.10 on Homebrew, mind you! Seems to have no problem on Python 3.10, so I've done that.